handsontable/types/focusManager.d.ts

```diff
@@ -0,0 +1,12 @@
+type FocusMode = 'cell' | 'mixed';
+
+export interface FocusManager {
```

Comment:
The class is exported using default export and the type is exported using named export. It's worth to unify the export styles. I'd prefer named exports.